stringutils.go

```diff
@@ -138,8 +139,27 @@ func getHarf(char rune) Harf {
 	return Harf{Unicode: char, Isolated: char, Medium: char, Final: char}
 }
 
+//removeAllNonAlphabetChars deletes all character which are not included in Arabic Alphabet
+func removeAllNonArabicChars(text string) string {
```

Comment:
Make the function name starts with an uppercase letter so it's exported for use outside the package.